isis/src/control/apps/autoseed/autoseed.h

```diff
@@ -0,0 +1,19 @@
+#ifndef autoseed_h
+#define autoseed_h
+
+#include "ControlNet.h"
+#include "Pvl.h"
+#include "UserInterface.h"
+
+enum SeedDomain {
+  XY,
+  SampleLine
+};
+
+namespace Isis {
```

Comment:
Should the `SeedDomain` enum be in the Isis namespace?

Reply:
Potentially? I'm not really sure when something should or should not be in the Isis namespace

Reply:
As a general rule-of-thumb everything we define should be in the Isis namespace to avoid conflicts with other packages. You basically never want something defined outside of a namespace in a header file.